tsconfig.json

```diff
@@ -3,7 +3,7 @@
         "baseUrl": ".",
         "downlevelIteration": true,
         "emitDecoratorMetadata": true,
-        "esModuleInterop": false,
+        "esModuleInterop": true,
```

Comment:
what is causing this change?

Reply:
this line https://github.com/okta/okta-angular/pull/81/files#diff-29999036b32d5fa366e290b1bbe22c692936fc2931de2a4bb4c2bba8906a2a74R24

this module is a commonjs module.

Reply:
Maybe let me try to find an ESM compatible package..